graphcanvas/graph_container.py

```diff
@@ -94,16 +94,22 @@ def _apply_graphviz_layout(layout):
                 component.x = self.width * (1 + layout[component._key][1])/2
 
         elif self.style == 'circular':
-            layout = networkx.drawing.nx_agraph.pygraphviz_layout(
-                self.graph, prog='twopi'
-            )
+            try:
+                layout = networkx.drawing.nx_agraph.pygraphviz_layout(
+                    self.graph, prog='twopi'
+                )
+                _apply_graphviz_layout(layout)
+            except ImportError:
```

Comment:
Strange indeed, but it is because `pygraphviz` and `Graphviz` are only optional dependencies for `networkx`, so the import doesn't occur until inside the function call [here](https://github.com/networkx/networkx/blob/networkx-1.11/networkx/drawing/nx_agraph.py#L254-L258).